### PR TITLE
Feature: NavBar 

### PR DESCRIPTION
**Pull request by ssimmons05**
## A short description of what this PR does.
Create routes for each of the pages.

## How was it tested?
This was linted with Yarn and tested with Jest.
router_test_screenshot

### Yarn Lint
router_lint_screenshot

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: src/router/Router.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import NavBar from "../components/NavBar";
+import Information from "../pages/Information";
+import Profile from "../pages/Profile";
+import FAQ from "../pages/FAQ";
+import Registration from "../pages/Registration";
+
+const Router = () => (
+  <BrowserRouter basename="/HacktoberFest">
+    <div className="App">
```
Comment (dylhack) [inline src/router/Router.jsx:11]: nit: if we want to be able to style this navbar even more we should add NavBar to the className.

**Review by dylhack — CHANGES_REQUESTED** [review]
Only one nitpick from me, other than that it lgtm.

File: src/router/Router.jsx (hunk 1 above)
Comment (ssimmons05) [reply src/router/Router.jsx:11]: Made this change.  You can see this here:  ff04b35d77ede341f74b9ca923718e9adefff20d

**Review by dylhack — APPROVED** [review]
LGTM

**Pull request merged**